src/ServiceManagement/ExpressRoute/ExpressRouteManagement/Microsoft.WindowsAzure.Management.ExpressRoute.nuget.proj

```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.ExpressRoute
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.ExpressRoute">
-      <PackageVersion>0.18.14-preview</PackageVersion>
+      <PackageVersion>0.19.14-preview</PackageVersion>
```

Comment:
@dihan0604 if you bump up minor version, patch version should be 0. should be 0.19.0